### DISCUSSION TITLE
Teamviewer not open

### DISCUSSION THREAD

**Discussion by hasukomusicvpn** [prasanthrangan/hyprdots#94] [Q&A]
I install Teamview via aur and follow the tutorials in https://www.youtube.com/watch?v=7rE3_3o5ccQ&ab_channel=MikeUrbizo
I can see teamviewer but it's not gonna open

**Comment by prasanthrangan** [comment]
Hi, aur packages may not always work, can you try installing it as flatpak?

**Reply by hasukomusicvpn** [reply]
> Hi, aur packages may not always work, can you try installing it as flatpak?

Flatpak doenst have teamviewer, maybe it's impossible to remote desktop on hyprland, thank you for your time, btw how can i access the wallpaper files

**Comment by kRHYME7** [comment]
RDP is not yet supported in wlroots as of now

**Comment by Enigma1309** [comment]
instead of team viewer you can use rustdesk, it works

**Reply by RookiexCookie** [reply]
Howd you got that to work?
I tried The normal version and Nightly build both gave me different errors and didnt connect

**Reply by Enigma1309** [reply]
what were the errors?
can you post here

**Reply by RookiexCookie** [reply]
I got this error from the normal build on my phone 
![Screenshot_20240506-152751](https://github.com/prasanthrangan/hyprdots/assets/136296017/8e3b2229-5ba8-4142-bfa2-ce73194fee09)

and this from nightly build 

![Screenshot_20240506-152908](https://github.com/prasanthrangan/hyprdots/assets/136296017/8aec4632-c78f-43dd-883b-6a3067260a73)

btw im using appimage

**Reply by kRHYME7** [reply]
Is the RDP protocol available now on wlroots based compositors?
We can RD control to other PCs but it's impossible to do the opposite. Or I'm missing something now.

If rdp protocol is now available, I can now use all the features of KDE connect 🥺

**Reply by Enigma1309** [reply]
@kRHYME7 yes it's available (at least works for me)
i've configured my friend's arch installation via rdp using Rust desk..
also not all features of kde connect work (ex:- remote input) but most features do work out of the box..

@RookiexCookie please try the aur version of rustdesk-bin 
it worked in my case without error
![image](https://github.com/prasanthrangan/hyprdots/assets/94917588/462f8201-945d-42f6-97d3-61cc080fbbdf)
also it will show an Wayland warning but it should work
